dydx/client.py

```diff
@@ -0,0 +1,87 @@
+import requests
+
+
+class Client(object):
+
+    BASE_API_URI = 'https://api.dydx.exchange/v1/dex/'
+
+    def __init__(self):
+        self.session = self._init_session()
+
+    # ------------ Helper Methods ------------
+
+    def _init_session(self):
+        session = requests.session()
+        session.headers.update({
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+            'User-Agent': 'dydx/python'
+        })
+        return session
+
+    def _request(self, method, uri, **kwargs):
+        response = getattr(self.session, method)(uri, **kwargs)
+        if not str(response.status_code).startswith('2'):
+            raise Exception(response)
+        return response.json()
+
+    def _get(self, *args, **kwargs):
+        return self._request('get', *args, **kwargs)
+
+    def _post(self, *args, **kwargs):
+        return self._request('post', *args, **kwargs)
+
+    def _put(self, *args, **kwargs):
+        return self._request('put', *args, **kwargs)
+
+    def _delete(self, *args, **kwargs):
+        return self._request('delete', *args, **kwargs)
+
+    # ------------ Public API ------------
+
+    def get_orders(self, **params):
+        """
+        Return all open orders for an address
+        :param makerAccountOwner: required
+        :type makerAccountOwner: string
```

Comment:
FYI I'm going to add two new params here on `get_orders` and `get_fills`
`limit` (number), and `startingAfter` (ISO-8601 string) for pagination